### ISSUE TITLE
how to can show bar value on bar?

### ISSUE THREAD

**Issue by iranandroid** [diogobernardino/williamchart#267]
Hi everyone
i wanna to show bar value top of every bar. is it possible?

**Comment by chriswiesner** [comment]
afaik it was possible in v2, but it's not supported for v3.

I'd also need to display values on top of the bars

**Comment by siavash-sajjad** [comment]
A progress on this task?

**Comment by siavash-sajjad** [comment]
@diogobernardino 
hi, can you guide me for add this feature?
i wanna publish my app, but i cann't find nice chart like your chart

**Comment by diogobernardino** [comment]
I recently introduced tooltips which could be added as complementary dependencies (e.g. `tooltip-slider:3.7.0`). I would expect this feature to be delivered as a tooltip too, but this is a just  guess for now.

**Comment by cacato86** [comment]
Hey @diogobernardino Any news on that? Interested to show the value of each bar too.

**Comment by diogobernardino** [comment]
As mentioned in my previous comment, I expect this to be very similar to the implementation of the slider tooltip. I would encourage to look at that particular implementation and develop something similar which best fits your needs.

**Comment by cacato86** [comment]
I'm taking a look, but as far as I know, tooltip only works when you click on a bar (or at least is the only way that I've seen where you have the value of each bar). Have I missed something?

Thank you anyways for your great job.

**Comment by diogobernardino** [comment]
>Have I missed something?

No, I think you're right, I might have misunderstood the feature. Perhaps it's worth having some examples to look at.

**Comment by OEssHRERLr770rGGLOH0** [comment]
They probably mean something like below. Label at the bottom, values at the top.


![](https://miro.medium.com/max/700/1*vdqF2by5yvcoPZRBTRHGOw.jpeg)

**Comment by joel-apollo** [comment]
I almost had this working for tooltips, but it doesn't quite work on the current interface. See https://github.com/diogobernardino/williamchart/issues/298
